schemas/org.gnome.shell.extensions.dash-to-dock.gschema.xml

```diff
@@ -47,7 +47,7 @@
   </enum>
   <schema path="/org/gnome/shell/extensions/dash-to-dock/" id="org.gnome.shell.extensions.dash-to-dock">
     <key name="dock-position" enum="org.gnome.shell.extensions.dash-to-dock.position">
-      <default>'LEFT'</default>
+      <default>'BOTTOM'</default>
```

Comment:
Why is this default changed?

Reply:
I'm not the author, but this is default dock position in GNOME 40

Reply:
Having the dock on the left/right side is a little bit problematic at the moment because it would cover the left/right workspace in the activities view. That's why someone suggested to make the position depenent on whether the activities overview is opened or not ([https://github.com/micheleg/dash-to-dock/issues/1431](https://github.com/micheleg/dash-to-dock/issues/1431)). In the activities overview the bottom is really the only place where the dash makes sense but on the desktop the left/right side is fine.

Reply:
Yeah, I'm handling this right now... Something is already in https://github.com/3v1n0/dash-to-dock/commits/gnome-40 but not updated with the latest bits yet, as I'm actually more keen to fix the math in gnome-shell upstream first and then apply the same to the extension via monkey-patching

Reply:
My opinion on this is that Dash to Dock should honor the GNOME Shell defaults and be a seamless "transition". If users want to restore the left-aligned dock the preference still exists and if they'd previously indicated a specific position it wouldn't be lost in the update.

Reply:
> Something is already in https://github.com/3v1n0/dash-to-dock/commits/gnome-40 but not updated with the latest bits yet, as I'm actually more keen to fix the math in gnome-shell upstream first and then apply the same to the extension via monkey-patching

I'd agree with this. The TLDR if I recall correctly is that GNOME Shell is using the work area, so right now this extension just removes the dock from chrome when in overview (if autohide is disabled). It is a very half-baked solution, it "works" :tm: but I'd definitely prefer to have the Shell do better and patch in that fix here. Someone did mention that a Shell commit already fixed some aspect ratio issues.

Reply:
Ok, the upstream fix is at https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/1892, will be used by this extension earlier though.